neo4j/resultsummary.go

```diff
@@ -44,8 +44,10 @@ const (
 type ResultSummary interface {
 	// Server returns basic information about the server where the statement is carried out.
 	Server() ServerInfo
-	// Statement returns statement that has been executed.
+	// Deprecated: Use Query instead
```

Comment:
shall we say when it's gonna be removed? (5.0, right?)

Reply:
Sounds like a good idea. And yes, 5.0 seems the right time to remove it.